package/kernel/linux/modules/netdevices.mk

```diff
@@ -1348,3 +1348,18 @@ define KernelPackage/sfc-falcon/description
 endef
 
 $(eval $(call KernelPackage,sfc-falcon))
+
+define KernelPackage/aqc
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=Aquantia AQtion (AQC107) 10Gbps Ethernet NIC
+  DEPENDS:=@PCI_SUPPORT +kmod-ptp +kmod-hwmon-core
```

Comment:
The CONFIG_AQUANTIA_PHY driver does not depend on any of this. Do you want to pack the `CONFIG_AQTION` driver?

Reply:
`+kmod-ptp +kmod-hwmon-core` you mean these? @hauke ?

Reply:
Please check again.

Reply:
BTW I'm not sure if I've undertood:
>Do you want to pack the CONFIG_AQTION driver?

Yes indeed I do.

Reply:
Then you have to pass the config option, I doubt you want the AQR PHY driver here

Reply:
@robimarko - what am I not getting here?

```shell
find . -iname *.ko | grep aq
  # ./build_dir/target-x86_64_musl/linux-x86_64/linux-5.10.134/drivers/net/phy/aquantia.ko
```

Reply:
What you want for AQC107 is `CONFIG_AQTION`